core/src/Streamly/Internal/Data/Unboxed.hs

```diff
@@ -290,6 +294,57 @@ class Unbox a where
         Int -> MutableByteArray -> a -> IO ()
     pokeByteIndex i arr = genericPokeByteIndex arr i
 
+data UnboxVSize a
```

Comment:
Move to a separate module `Streamly.Data.Serialize`. Rename this to `Serialize`.